README.md

```diff
@@ -181,7 +181,8 @@ $adapter = new DoctrineDbalSingleTableAdapter($queryBuilder, $countField);
 
 ### DoctrineORMAdapter
 
-To paginate [DoctrineORM](http://www.doctrine-project.org/projects/orm) query objects.
+To paginate [DoctrineORM](http://www.doctrine-project.org/projects/orm) query objects. By default `DoctrineORMAdapter` 
+won't work with single tables. Second parameter has to be set to `false` to make it work with single tables.
```

Comment:
this statement is wrong. It works, but not in the optimal way (which is precisely why the default value is ``true``, as it is better to work only but slower in some cases than to be broken in many cases by default)